.coderabbit.yaml

```diff
@@ -0,0 +1,13 @@
+language: en-US
+early_access: true
+reviews:
+  high_level_summary: false
+  review_status: false
+  poem: false
+  collapse_walkthrough: true
```

Comment:
Can this instead be turned off and only included when requested? It adds a lot of clutter to emails.

Reply:
That's a good suggestion. Will update.